plugins/code-coverage-backend/README.md

```diff
@@ -2,6 +2,50 @@
 
 This is the backend part of the `code-coverage` plugin. It takes care of processing various coverage formats and standardizing them into a single json format, used by the frontend.
 
+## Installation
+
+`yarn workspace backend add @backstage/plugin-code-coverage-backend`
+
+Next you will need to add code to setup the plugins router, in `packages/backend/src/plugins/codecoverage.ts` add the following:
+
+```typescript
+import { createRouter } from '@backstage/plugin-code-coverage-backend';
+import { Router } from 'express';
+import { PluginEnvironment } from '../types';
+
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  return await createRouter({
+    config: env.config,
+    discovery: env.discovery,
+    database: env.database,
+    urlReader: env.reader,
+    logger: env.logger,
+  });
+}
+```
+
+Finally we need to load the plugin in `packages/backend/src/index.ts`, add the following:
+
+```typescript
+import codeCoverage from './plugins/codecoverage';
+```
+
+create the env for code coverage:
+
+```typescript
+const codeCoverageEnv = useHotMemoize(module, () =>
+  createEnv('code-coverage'),
+);
+```
+
+Then inform the router for the path:
```

Comment:
This is another case where being more detailed is better. I would mention that this should go along with all the other `apiRouter.use()` statements. I've seen cases where people but this right after the `createEnv` portion above